common/src/main/kotlin/io/homeassistant/companion/android/database/settings/Setting.kt

```diff
@@ -12,5 +12,7 @@ data class Setting(
     @ColumnInfo(name = "websocket_setting")
     var websocketSetting: WebsocketSetting,
     @ColumnInfo(name = "sensor_update_frequency")
-    var sensorUpdateFrequency: SensorUpdateFrequencySetting
+    var sensorUpdateFrequency: SensorUpdateFrequencySetting,
+    @ColumnInfo(name = "push_provider", defaultValue = "FCM")
+    var pushProvider: PushProviderSetting
```

Comment:
```suggestion
    var pushProvider: PushProviderSetting,
```
It will make future update of this class easier to review